.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow is meant to trigger a build of Docker binaries when a tag
+# is created, it uses the existing `build-docker-image.yml` workflow
+#
+# We use a separate action as we might want to trigger this with different under
+# different circumstances than a Continous Deployment, for example.
+name: Release binaries
+
+on:
+  create:
+
+jobs:
+  # Each time this workflow is executed, a build will be triggered to create a new image
+  # with the corresponding tags using information from git
+  #
+  # The image will be commonly named `zebrad:<short-hash | github-ref | semver>`
+  build:
+    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
If we decide to remove the tag filter, we'll want to remove it here as well.

Reply:
Fixed with [ci(release): just publish to DockerHub when a release is published](https://github.com/ZcashFoundation/zebra/pull/5138/commits/53191eaea9016ac8b0cd7633d77ed610fa9be62c)